src/components/smart_objects/src/object_schema_item.cc

```diff
@@ -251,25 +252,23 @@ CObjectSchemaItem::CObjectSchemaItem(const Members& members)
 
 void CObjectSchemaItem::RemoveFakeParams(
     SmartObject& Object, const utils::SemanticVersion& MessageVersion) {
-  for (SmartMap::const_iterator it = Object.map_begin();
-       it != Object.map_end();) {
+  for (SmartMap::const_iterator it = Object.map_begin(); it != Object.map_end();
```

Comment:
@YarikMamykin I would simplify loop to `for (const auto& key : Object.enumerate()) {}`

Reply:
@mkorniichuk , fixed in 29d5f70ba69b90cc1f4bfcb30cdc4fbe7028e7a4